test/gpu/native/studies/shoc/shoc-sort.suppressif

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Suppress ROCm6 shoc-sort failures: https://github.com/chapel-lang/chapel/issues/26669
+check_conditions() {
+    chpl_gpu=$(printenv CHPL_GPU)
+    chpl_test_perf_label=$(printenv CHPL_TEST_PERF_LABEL)
+
+    # Check if CHPL_GPU is 'amd' and CHPL_TEST_PERF_LABEL is set 
+    if [[ "$chpl_gpu" != "amd" || -z "$chpl_test_perf_label" ]]; then
+        return 1
+    fi
+
+    chpl_home=$(printenv CHPL_HOME)
+    if [[ -z "$chpl_home" ]]; then
+        return 1
+    fi
+
+    # Check if CHPL_GPU_SDK_VERSION contains '6'
+    chpl_env_output=$("$chpl_home/util/printchplenv" --all)
+    if echo "$chpl_env_output" | grep -q 'CHPL_GPU_SDK_VERSION: 6'; then
```

Comment:
This is probably a bit loose (what will happen with 7.6? :) ), but should be OK.

Reply:
That'd be great news if 7.6 generated a "passing suppressif" mail!  Now I can't wait for 7.6!  :D

Reply:
It won't match `CHPL_GPU_SDK_VERSION: 7`

Reply:
The comment was misleading, the code makes sure it starts with `6`

Reply:
Sorry to rain on your excitement Brad :P

Reply:
Whoops, yeah. Sorry, I didn't read the code closely.